### PR TITLE
Upgrade to Project.toml

### PR DESCRIPTION
**Pull request by juliohm**
This PR replaces the old REQUIRE by the new Project.toml.

Should the version number increase to `v0.2.3` or directly to `v0.3.0`?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,13 +4,9 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
+  - 1.3
```
Comment (joshday) [inline .travis.yml:10]:
```suggestion
  - 1.0
  - 1.1
  - 1.2
  - 1.3
```

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name = "LearnBase"
+uuid = "553a3efa-3f4c-4899-931a-92df696021e0"
+version = "0.2.3"
+
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+
+[compat]
+StatsBase = "0.33"
```
Comment (joshday) [inline Project.toml:10]:
```suggestion
StatsBase = "0.33"
julia = "1"
```

**Review by joshday — APPROVED** [review]

**Comment by juliohm** [comment]
Thank you @joshday for merging. Should we trigger a new release as well?

We have TagBot already set as a GitHub action, and I wonder if we have JuliaRegistrator already configured for the organization? If yes, we could trigger it right away on master.

**Comment by joshday** [comment]
Yeah, that'd be great!

**Comment by juliohm** [comment]
First we need to double check the current version specified in the Project.toml. It seems like some packages are referring to LearnBase 0.3.0? What is the latest version before Project.toml?

**Comment by joshday** [comment]
Which ones?  Latest is 0.2.2 (https://github.com/JuliaML/LearnBase.jl/releases)

**Comment by juliohm** [comment]
For instance, LossFunctions.jl REQUIRE file is pointing to LearnBase 0.3.0: https://github.com/JuliaML/LossFunctions.jl/blob/master/REQUIRE

**Comment by juliohm** [comment]
Any idea @joshday of the latest tagged version? Can we just trigger JuliaRegistrator and see?

**Pull request merged**